example_kubeflow_pipelines/openvino/container/slim_model.py

```diff
@@ -0,0 +1,216 @@
+import tensorflow as tf
+from tensorflow.python.saved_model import signature_constants
+from tensorflow.python.saved_model import tag_constants
+from nets import nets_factory
+from tensorflow.python.platform import gfile
+import argparse
+import validators
+import os
+import requests
+import tarfile
+from subprocess import Popen, PIPE
+import shutil
+import glob
+import re
+import json
+from tensorflow.python.tools.freeze_graph import freeze_graph
+from tensorflow.python.tools.saved_model_cli import _show_all
+from urllib.parse import urlparse
+from shutil import copyfile
+from google.cloud import storage
+
+
+def upload_to_gcs(src, dst):
+    parsed_path = urlparse(dst)
+    bucket_name = parsed_path.netloc
+    file_path = parsed_path.path[1:]
+    gs_client = storage.Client()
+    bucket = gs_client.get_bucket(bucket_name)
+    blob = bucket.blob(file_path)
+    blob.upload_from_filename(src)
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Slim model generator')
```

Comment:
it could be used. I think there is missing importing checkpoint to include the model weights before the export to saved_model. The process of conversion should be pretty universal.
slim_model.py works for models from https://github.com/tensorflow/models/blob/master/research/slim/nets/nets_factory.py